docs/tutorials/image_classification.md

```diff
@@ -8,11 +8,11 @@ In this tutorial we will learn how to:
 ## Setting up the Project
 
 First, create a new project directory named `image_classification` with the following structure:
-    
+
 ```plaintext
+config.yaml
```

Comment:
I think this is where the config is meant to live, according to the commands in the tutorial.